### PR TITLE
Bump Go in provisioner

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

**Related issue(s)**

### CODE REVIEW

File: resources/kcp/charts/provisioner/values.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ global:
     path: europe-docker.pkg.dev/kyma-project
   images:
     provisioner:
-      version: "v20230930-6fff193f"
-      dir: "prod"
+      version: "PR-3113"
```
Comment (koala7659) [inline resources/kcp/charts/provisioner/values.yaml:6]: we cannot update the image version here to the one taken from dev repo

**Review by koala7659 — CHANGES_REQUESTED** [review]
Please check my comment

**Review by koala7659 — APPROVED** [review]
LGTM

**Review by gczuczy — APPROVED** [review]

**Pull request merged**